.github/workflows/indigo-ci.yaml

```diff
@@ -929,7 +929,7 @@ jobs:
         run: git fetch --tags -f
       - name: Install postgres server headers
         run: |
-          case ${{ matrix.postgres_major_version }} in 12) pg_version=12;; 11) pg_version=11;; 10) pg_version=10;; 9.6) pg_version=96;; esac;

```

Comment:
This line can be left unchanged. Dropping 9.6 is provided by line#919.

Reply:
fixed